.github/workflows/test.yml

```diff
@@ -26,19 +40,6 @@ jobs:
       - name: Install Dependencies
         run: npm ci
 
-      - name: Run Unit Tests
-        run: |
-          npm run test:js:ci
```

Comment:
where would others tests reside if removed from here ?

Reply:
transformer tests covered in code coverage yaml

Reply:
Then why to remove the parallelism ?
Let the code coverage action run independently which covers execution of all the tests.
And this action - good with removing duplicate unit tests & restrict it only to UserTransformation tests & rename the file/action to ut-tests.

Reply:
for coverage workflow also we are running tests that's why we remove from this

Reply:
addressed